server/db/tables.js

```diff
@@ -81,10 +86,11 @@ const createInterventionTable = () => {
       console.log(err);
       pool.end();
     });
-}
+};
 
 /**
  * Drop Tables
+ * @returns {object} object
```

Comment:
Since this doesn't return anything, you can just use void. eg
```suggestion
 * @returns {void}
```